src/os/bluestore/bluefs_types.cc

```diff
@@ -282,3 +282,11 @@ ostream& operator<<(ostream& out, const bluefs_transaction_t& t)
 	     << " crc 0x" << t.op_bl.crc32c(-1)
 	     << std::dec << ")";
 }
+
+uint64_t bluefs_transaction_t::expected_size() const {
```

Comment:
done

Reply:
Sorry, perhaps I haven't shared my point clear enough. Apart from renaming I'd prefer to see the implementation of this method via denc methods to match encode/decode rather than using questionable sizeof(var) constructs.